app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -11,5 +11,5 @@ class VerifyCsrfToken extends Middleware
      *
      * @var array<int, string>
      */
-    protected $except = ['/login', '/logout', '/signup'];
+    protected $except = ['/login', '/logout', '/signup', '/api/learnings-count'];
```

Comment:
- `csrf` can be entirely disabled. just use asterisk, so you don't need to append in each route you add.
>api routes are stateless and therefore cannot track any issued token
csrf is not required or possible in any of the api.php routes
https://laracasts.com/discuss/channels/laravel/how-to-disable-csrf-protection-on-api-routes-when-using-passport-and-javascript?page=1&replyId=521277
```suggestion
    protected $except = ['*'];
```